ARE/services/WebService/src/main/java/eu/asterics/mw/webservice/SseResource.java

```diff
@@ -136,6 +139,7 @@ public EventOutput subscribe_modelStateEvents() {
     @Path("/runtime/model/channels/event/listener")
     @GET
     @Produces(SseFeature.SERVER_SENT_EVENTS)
+    @Description("Subscribes for model events. The events are delivered using SSE.")
```

Comment:
The descriptions of `/runtime/model/state/listener` and `/runtime/model/channels/event/listener` do not really explain or emphasize the difference between both paths. My suggestion:

```java
@Description("Subscribe for model state events.")
```

```java
@Description("Subscribe for model channel events.")
```

Reply:
👍 
however as mentioned above maybe it makes sense to mention SSE in the description.